benchmarks/recursive_computation/config.yaml

```diff
@@ -0,0 +1,50 @@
+shallow: &base
+  max_depth: 10
+  enable_sleep: false
+  sleep_duration: 0.1
+  nspans: 1
+  profiler_enabled: false
+
+medium: 
+  <<: *base
+  max_depth: 50
+
+deep:
+  <<: *base
+  max_depth: 100
+
+deep-with-spans:
+  <<: *base
+  max_depth: 100
+  nspans: 5
+
+intensive:
+  <<: *base
+  max_depth: 50
+  nspans: 3
+
+sleep-scenario:
+  <<: *base
+  max_depth: 20
+  enable_sleep: true
+  sleep_duration: 0.01
+
+shallow-profiled:
+  <<: *base
+  profiler_enabled: true
+
+medium-profiled:
+  <<: *base
+  max_depth: 50
+  profiler_enabled: true
+
+deep-profiled:
+  <<: *base
+  max_depth: 100
```

Comment:
how many stack frames would you expect to see in the flamegraph with `max_depth:100`?

Reply:
so, it'd be 200 right? we want to test a scenario where we're unwinding close to the limit, 512. Mind increasing this?